api/RTCRtpEncodingParameters.json

```
@@ -210,7 +210,7 @@
           "spec_url": "https://w3c.github.io/webrtc-extensions/#dom-rtcrtpencodingparameters-maxframerate",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "83"
```

@bc-lee this now says 83 instead of 81. I see that https://chromestatus.com/feature/6216116758118400 says 83, but I don't think that's right.

```suggestion
              "version_added": "81"
```